src/components/Tabs/index.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react'
 import { Tab, Row, Col, Nav } from 'react-bootstrap'
-import LogOut from '../LogOut'
```

Comment:
Don't we need this ?

Reply:
I included Dilip's navbar in this component and it includes the LogOut component. It looks better that way.

Reply:
I'll close this pr because I just recognized I've been working on this same branch now on redux.